src/conversion/model-to-view-converters.js

```diff
@@ -288,6 +289,157 @@ export function unwrap( elementCreator ) {
 }
 
 /**
+ * Function factory, creates a converter that converts new model marker to view attribute element.
+ *
+ * Let's assume following model and view. `{}` represents a range that is added as a marker with `searchResult` name.
+ * The range represents a result of search `ab` string in the model document. The range has to be visualized in view.
+ *
+ *		[paragraph]              MODEL ====> VIEW        <p>
+ *			|- {a                                         |- <span class="searchResult">
+ *			|-  b}                                        |   |- ab
+ *			|-  c                                         |- c
+ *
+ * The wrapping node depends on passed parameter. If {@link engine.view.AttributeElement} was passed, it will be cloned and
+ * the copy will become the wrapping element. If `Function` is provided, it is passed all the parameters of the
+ * {@link engine.conversion.ModelConversionDispatcher#event:addMarker addMarker event}. It's expected that the
+ * function returns a {@link engine.view.AttributeElement}. The result of the function will be the wrapping element.
+ * When provided `Function` does not return element, then will be no conversion.
+ *
+ * The converter automatically consumes corresponding value from consumables list, stops the event (see
+ * {@link engine.conversion.ModelConversionDispatcher}).
+ *
+ *		modelDispatcher.on( 'addMarker:searchResult', wrapMarker( new ViewAttributeElement( 'span', { class: 'searchResult' } ) ) );
+ *
+ * @external engine.conversion.modelToView
+ * @function engine.conversion.modelToView.wrapMarker
+ * @param {engine.view.AttributeElement|Function} elementCreator View attribute element, or function returning
+ * a view attribute element, which will be used for wrapping.
+ * @returns {Function} Add marker converter.
+ */
+export function wrapMarker( elementCreator ) {
```

Comment:
What is the difference between `wrap` and `wrapMarker`? From the view point of view, there is nothing like `marker` to be wrapped.